Makefile.am

```diff
@@ -80,7 +80,8 @@ all-local: digraphs.la
 	$(mkdir_p) $(top_srcdir)/$(BINARCHDIR) $(top_srcdir)/bin/lib
 	cp -RL @PLANARITY_SUITE_DIR@/.libs/*  $(top_srcdir)/bin/lib/
 if SYS_IS_CYGWIN
-	cp .libs/digraphs.dll $(GAPINSTALLLIB)
+	cp .libs/digraphs.dll $(GAPINSTALLLIB)/digraphs.so
+	cp bin/lib/cygplanarity-0.dll $(GAPINSTALLLIB)
```

Comment:
@embray Is this what you meant?

Reply:
Yes, I see that now.  And yes, I think that should work.

However, I also don't think it's good to rename the .dll file to .so.  Maybe it's fine for now if it's needed as a workaround, but ideally GAP should be fixed to use the right filename extension for loading dynamic libs.